donate.html

```diff
@@ -678,6 +730,50 @@ <h5 class="margin-bottom-normal">{% filter ak_text:"donate_billing_information"
   });
 
   function change_step_one_label(current_step) {
+    if (current_step == "1") {

```

Comment:
Event documentation for reference:
https://developers.google.com/analytics/devguides/collection/ua/gtm/enhanced-ecommerce#add